res/linuxPackaging.json

```diff
@@ -18,8 +18,8 @@
     "x-scheme-handler/ftp"
   ],
   "requires": [
-    "GConf2",
-    "libXScrnSaver",
+    "(GConf2 or gconf2)",
+    "(libXScrnSaver or libXss1)",
```

Comment:
Nothing has changed for Leap:
```
Problem: nothing provides (GConf2 or gconf2) needed by brave-0.20.0-1.x86_64
 Solution 1: do not install brave-0.20.0-1.x86_64
 Solution 2: break brave-0.20.0-1.x86_64 by ignoring some of its dependencies

Choose from above solutions by number or cancel [1/2/c] (c):
```

Considering this issue is 6 months old I still think fixing it for Tumbleweed is better than having it broken on both.